integration-cli/requirements.go

```diff
@@ -32,6 +33,13 @@ var (
 		func() bool { return supportsExec },
 		"Test requires 'docker exec' capabilities on the tested daemon.",
 	}
+	Network = TestRequirement{
+		func() bool {
+			_, err := http.Get("http://google.com")
```

Comment:
Not at all -- if we use "--net=host", for example, vanity network interface
names are a very real possibility (like "lan0" here on my local box).